pkg/netpol/eval/internal/k8s/namespace.go

```diff
@@ -16,6 +17,10 @@ import (
 type Namespace struct {
 	Name   string
 	Labels map[string]string
+	// RepresentativeNsLabelSelector points to the namespaceSelector of the policy rule which this
+	// representative namespace was inferred from
+	// used only with representative peers (exposure-analysis)
+	RepresentativeNsLabelSelector *v1.LabelSelector
```

Comment:
why is this part of the namespace object, and not as part of the representative pod object?

Reply:
moved to the pod object